test/account_multisig.test.js

```diff
@@ -82,6 +82,17 @@ describe('deployMultisig key rotations', () => {
 
 describe('account2fa transactions', () => {
 
+    test('add app key before deployMultisig', async() => {
+        let account = await testUtils.createAccount(nearjs);
+        const appPublicKey = KeyPair.fromRandom('ed25519').getPublicKey();
+        const appMethodNames = ['some_app_stuff','some_more_app_stuff'];
+        await account.addKey(appPublicKey.toString(), 'foobar', appMethodNames.join(), new BN(parseNearAmount('0.25')));
```

Comment:
not sure why `.join()` call is needed on `appMethodNames`. Same about `.toString()` BTW.

Reply:
odd behavior with near-api-js returning arrays that don't deepEqual

Reply:
@mattlockyer but you basically end up testing adding a key for one method_name with comma inside instead of testing multiple method names. This is definitely not what is desired.

Reply:
Needs 436 I think?